core/domain/feedback_domain.py

```diff
@@ -19,7 +19,22 @@
 
 
 class FeedbackThread(object):
-    """Domain object for a feedback thread."""
+    """Domain object for a feedback thread.
+    Attributes:
+        full_thread_id: An int representing the feedback thread ID.
```

Comment:
See formatting in e.g. #2465. It should be:

```
    full_thread_id: int. The feedback thread ID.
```

and similar, for all the other attributes, arg names, etc. Please correct this, here and elsewhere.

Also, are you sure this is an int? How do you know?